public/en/releases.html

```diff
@@ -0,0 +1,373 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>io.js - JavaScript I/O</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <meta name="description" content="io.js is an npm compatible platform originally based on node.js">
+  <meta name="keywords" content="iojs, io.js, io js, javascript io, uv, libuv, node-forward, node forward, node, node.js, node.js forward, nodejs, nodejs forward, javascript">
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet">
+  <link href="../main.css" rel="stylesheet">
+  <link href="../mobile.css" rel="stylesheet">
+  <link href="../releases.css" rel="stylesheet">
+  <link rel="icon" href="/images/1.0.0.ico" type="image/x-icon">
+  <link rel="apple-touch-icon" href="/images/apple-touch-icon-1.0.0.png">
+  <meta property="og:image" content="https://iojs.org/images/1.0.0.png">
+  <meta property="og:image:type" content="image/png">
+  <meta property="og:image:width" content="1369">
+  <meta property="og:image:height" content="1563">
+</head>
+
+<body markdown-page="releases">
+
+  <header>
+    <div class="content">
+      <a href="index.html" class="logo">io.js</a>
+      <div class="spacer"></div>
+      <a href="faq.html">FAQ</a>
+      <a href="es6.html">ES6</a>
+      <a href="https://iojs.org/api/">API Docs</a>
+    </div>
+  </header>
+
+  <div class="content content--body clearfix">
+    <h1>io.js Release History</h1>
+
+<div class="releaseTable">
+  <div class="releaseTable-release">
+    <div class="releaseTable-versionRow">
+      <h2 class="releaseTable-version">
+        v1.5.1
```

Comment:
In a PR-or-two I'd love for each release to have a landing page. Perhaps a hybrid of the changelog (markdown based), automatic download/docs links, etc.

For now, ya, could be downloads. Although we do have an explicit link for them.